easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCC-4.8.2-no-cpp-patch.eb

```diff
@@ -1,11 +1,16 @@
 name = 'ncurses'
 version = '5.9'
+versionsuffix = '-no-cpp-patch'
```

Comment:
drop the versionsuffix, you're not changing the behaviour of the software at all here, so no versionsuffix is needed

oh, and don't change the name for existing files, just create a copy (which you won't need anymore now)